tools/build-tools/src/fluidBuild/tasks/workers/workerPool.ts

```diff
@@ -44,15 +45,21 @@ export class WorkerPool {
     public async runOnWorker(workerName: string, command: string, cwd: string): Promise<WorkerExecResultWithOutput> {
         const workerMessage: WorkerMessage = { workerName, command, cwd };
         const cleanup: (() => void)[] = [];
-        const installTemporaryListener = (object: EventEmitter, event: string, handler: any) => {
+        const installTemporaryListener = (object: EventEmitter | Readable, event: string, handler: any) => {
             object.on(event, handler);
             cleanup.push(() => object.off(event, handler));
         }
         const setupWorker = (worker: Worker | ChildProcess, res: (value: WorkerExecResultWithOutput) => void) => {
             let stdout = "";
             let stderr = "";
-            installTemporaryListener(worker.stdout, "data", (chunk: any) => { stdout += chunk; });
-            installTemporaryListener(worker.stderr, "data", (chunk: any) => { stderr += chunk; });
+
+            if (worker.stdout) {
+                installTemporaryListener(worker.stdout, "data", (chunk: any) => { stdout += chunk; });
+            }
+            if (worker.stderr) {
+                installTemporaryListener(worker.stderr, "data", (chunk: any) => { stderr += chunk; });
+            }
```

Comment:
@curtisman @heliocliu Do you know if this is a safe change?

Reply:
Yeah looks fine.  Guessing this was because the node types here was still on 11 instead of 12?